pennylane_lightning/core/src/simulators/lightning_kokkos/gates/GateFunctorsParam.hpp

```diff
@@ -1687,4 +1483,426 @@ template <class PrecisionT, bool inverse = false> struct rotFunctor {
     }
 };
 
+template <class PrecisionT> struct apply1QubitOpFunctor {
+
+    using ComplexT = Kokkos::complex<PrecisionT>;
+    using KokkosComplexVector = Kokkos::View<ComplexT *>;
+    using KokkosIntVector = Kokkos::View<std::size_t *>;
+
+    KokkosComplexVector arr;
+    KokkosComplexVector matrix;
+    const std::size_t n_wires = 1;
+    const std::size_t dim = 1U << n_wires;
```

Comment:
Is there implicit type conversion from `U` to `size_t` here?

Reply:
I don't think it does (or maybe depends on the compiler). Nice catch! Will change all similar casts.